### DISCUSSION TITLE
httpcore 0.15.0 support

### DISCUSSION THREAD

**Discussion by tbhi** [encode/httpx#2237] [Potential Issue]
https://github.com/encode/httpcore/releases/tag/0.15.0 was released recently but `setup.py` has `httpcore>=0.14.5,<0.15.0` - can this be bumped?

**Comment by michaeloliverx** [comment]
This would unblock https://github.com/encode/httpx/pull/2096

**Comment by lovelydinosaur** [comment]
Addressed in #2214 yup. (Which is waiting for review)

**Reply by michaeloliverx** [reply]
Should we try and get https://github.com/encode/httpx/pull/2096 in too or wait till another release?

**Reply by lovelydinosaur** [reply]
Not sure - do you want to try to get the test suite passing against that pull request first?

**Reply by michaeloliverx** [reply]
All green now buddy ✅
